docker/Dockerfile.apache

```diff
@@ -0,0 +1,26 @@
+FROM hathitrust-imgsrv
+
+RUN apt-get -y install apache2 libapache2-mod-fcgid
+RUN rm /etc/apache2/sites-available/*
+ADD ./docker/conf-available /etc/apache2/conf-available
+# ADD ./docker/sites-available /etc/apache2/sites-available
+COPY bin /opt/docker
+RUN chmod +x /opt/docker/*.sh
+
+RUN /usr/sbin/a2dismod 'mpm_*'
+RUN a2enmod headers
+RUN a2enmod env
+RUN a2enmod mpm_prefork
+RUN a2enmod rewrite
+RUN a2enmod proxy
+RUN a2enmod proxy_fcgi
+RUN a2enmod proxy_http
+RUN a2enmod cgi
+RUN a2enconf servername
+RUN a2ensite 000-default
```

Comment:
This ultimately fails for me -
```
 > [17/19] RUN a2ensite 000-default:
#0 0.301 ERROR: /etc/apache2/sites-enabled/000-default.conf is a dangling symlink!
#0 0.301 ERROR: Site 000-default does not exist!
```

Reply:
probably because we're removing the default sites-available but the `ADD ./docker/sites/available /etc/apache2/sites-available` above is commented out?